src/Keboola/CsvTable/Table.php

```diff
@@ -1,45 +1,58 @@
 <?php
 
+declare(strict_types=1);
+
 namespace Keboola\CsvTable;
 
 use Keboola\Csv\CsvOptions;
 use Keboola\Csv\CsvWriter;
 use Keboola\Temp\Temp;
+use UnexpectedValueException;
 
 /**
  * CsvFile class with attribute, primaryKey, incremental and name properties
  */
-class Table extends CsvWriter {
-	/** @var array */
-	protected $attributes = array();
-
-	/** @var array */
-	protected $primaryKey;
-
-	/** @var Temp */
-	protected $temp;
-
-	/** @var string */
-	protected $name;
-
-	/** @var bool|null */
-	protected $incremental = null;
-
-	/** @var array */
-	protected $header = [];
-
-	public function __construct(string $name, array $header = [], bool $writeHeader = true, Temp $temp = null, $delimiter = CsvOptions::DEFAULT_DELIMITER, $enclosure = CsvOptions::DEFAULT_ENCLOSURE, $lineBreak = "\n")
-    {
+class Table extends CsvWriter
+{
+    /**
+     * @var array
+     */
+    protected array $attributes = [];
+
+    /**
```

Comment:
`@var array` to být nemusí

Reply:
Jakto?
```
public function setPrimaryKey($primaryKey): void
    {
        if (!is_array($primaryKey)) {
            $primaryKey = explode(',', $primaryKey);
        }

        $this->primaryKey = $primaryKey;
    }
```

Reply:
tak už tam máš `protected array $primaryKey;`

Reply:
Jo myslíš, tu anotaci. Já myslel, že nesouhlasíš obecně s tím typem. Chápu :)